### PR TITLE
Docfix(grammar and sentence structure) and .gitattributes polish

### PR DESCRIPTION
**Pull request by 7k5x**
The documentation is kinda erred in some places and I smoothed them out.<br>The .gitattributes file has additions so the user will have only the essential stuff. (Sorry I typed this out on an iPod)

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -1,3 +1,7 @@
 /.gitattributes export-ignore
 /.gitignore export-ignore
 /.github export-ignore
+/CODE_OF_CONDUCT.md export-ignore
+/README.org export-ignore
+/LICENSE export-ignore
```
Comment (ojaha065) [inline .gitattributes:6]: Just something to think about:
Generally, copyright notice should be included in all downloadable archives. I don't know if that's specially required by MPL 2.0 license, but when it comes to copyright stuff, I think it's better be safe than sorry.

Comment (7k5x) [reply .gitattributes:6]: I guess you're right. I don't have a good sense of licensing, so thanks for helping me out!

Comment (7k5x) [reply .gitattributes]: I updated `.gitattributes` . Are there any more violations?

File: .gitattributes [hunk 2]
```diff
@@ -1,3 +1,6 @@
 /.gitattributes export-ignore
 /.gitignore export-ignore
 /.github export-ignore
+/CODE_OF_CONDUCT.md export-ignore
+/README.org export-ignore
+/CREDITS export-ignore
```
Comment (black7375) [inline .gitattributes:6]: Similarly, for copyright reasons, I think it would be better to include the `CREDITS` file.

File: README.org [hunk 3]
```diff
@@ -116,16 +116,16 @@
...
-  Or =about:config='s =svg.context-properties.content.enabled= to =true=
+  Please follow [[https://github.com/black7375/Firefox-UI-Fix/wiki/Installation-Guide][Installation Guide]], \\
+  or set =about:config='s =svg.context-properties.content.enabled= to =true= .
 
   - *Some panel menu items doesn't show* \\
     [[https://user-images.githubusercontent.com/25581533/120487528-93b40200-c3a5-11eb-98ad-3498beb9f38e.png]]
 
-  Please follow [[https://github.com/black7375/Firefox-UI-Fix/wiki/Installation-Guide][Installation Guide]]. \\
-  Or =icons= directory copy.
+  Please follow the [[https://github.com/black7375/Firefox-UI-Fix/wiki/Installation-Guide][Installation Guide]], \\
```
Comment (black7375) [inline README.org:130]: Would it be better to have `the`?

**Comment by black7375** [comment]
Overall I like it.

File: README.org (hunk 3 above)
Comment (7k5x) [reply README.org:130]: thats a typo.. The commit is an addition of ‘on’.

**Comment by 7k5x** [comment]
@black7375 Please review this PR again. I have fixed the credits part.

File: README.org (hunk 3 above)
Comment (black7375) [reply README.org:130]: What do you mean?

File: README.org [hunk 4]
```diff
@@ -116,16 +116,16 @@
...
 
-** F&A
+** FAQ
 
   - *Black pixels around the selected tab bottom corners* \\
     [[https://user-images.githubusercontent.com/5571586/120401980-edf58a00-c2f5-11eb-9e64-ce50c5b189b2.png]]
 
-  Please follow [[https://github.com/black7375/Firefox-UI-Fix/wiki/Installation-Guide][Installation Guide]]. \\
-  Or =about:config='s =svg.context-properties.content.enabled= to =true=
+  Please follow [[https://github.com/black7375/Firefox-UI-Fix/wiki/Installation-Guide][Installation Guide]], \\
```
Comment (black7375) [inline README.org:124]: There is no `the` here, so I asked.

**Comment by 7k5x** [comment]
@black7375 No it's not `the` that I added. 
image
I was talking about this. `It works on FF 89 and above` is the correct sentence. There is a typo on the commit. Should I open a new PR for no typos?

**Comment by black7375** [comment]
No, it was added.

Please check `Files changed` menu.
image

**Comment by 7k5x** [comment]
@black7375 oh ok you mean that? I think adding `the` makes more sense. `Please follow *the* Installation Guide` is more proper than `Please follow Installation Guide` .

**Comment by black7375** [comment]
`the` shuould be entered in the same way here https://github.com/black7375/Firefox-UI-Fix/pull/84#discussion_r650341301.
Documentation should be consistent.

File: README.org [hunk 5]
```diff
@@ -116,16 +116,16 @@
   - Proton
   - Lepton
 
-** F&A
+** FAQ
```
Comment (black7375) [inline README.org:119]: TOC should be updated.

**Review by black7375 — CHANGES_REQUESTED** [review]

File: README.org (hunk 5 above)
Comment (7k5x) [reply README.org:119]: @black7375 What does TOC mean? I edited F&A to FAQ because it’s more frequently used.

**Comment by 7k5x** [comment]
@black7375 I added a the for consistency. What should be fixed now? You always seem to be very reluctant to merge my PRs.

**Comment by black7375** [comment]
I am not deliberately ostracizing it.

File: README.org (hunk 5 above)
Comment (black7375) [reply README.org:119]: TOC is Table Of Contents
https://github.com/black7375/Firefox-UI-Fix#lepton-old-name-proton-fix

`[[#fa][F&A]]` => `[[#faq][FAQ]]`

Comment (7k5x) [reply README.org:119]: OK. I’ll fix it right away.

**Comment by black7375** [comment]
Now it looks good.

**Comment by 7k5x** [comment]
Not really. I know I’m not very calm and meticulous. Sorry if you’re annoyed.

**Pull request merged**